Packs/FeedAutofocus/Integrations/FeedAutofocusDaily/FeedAutofocusDaily.yml

```diff
@@ -0,0 +1,110 @@
+category: Data Enrichment & Threat Intelligence
+commonfields:
+  id: AutoFocus Daily Feed
+  version: -1
+configuration:
+- display: Fetch indicators
+  name: feed
+  required: false
+  defaultvalue: true
+  type: 8
+- display: The AutoFocus API key
+  name: api_key
+  required: true
+  type: 4
+- additionalinfo: Indicators from this integration instance will be marked with this reputation
+  defaultvalue: Bad
+  display: Indicator Reputation
+  name: feedReputation
+  options:
+    - None
+    - Good
+    - Suspicious
+    - Bad
+  required: false
+  type: 18
+- additionalinfo: Reliability of the source providing the intelligence data
+  defaultvalue: A - Completely reliable
+  display: Source Reliability
+  name: feedReliability
+  options:
+  - A - Completely reliable
+  - B - Usually reliable
+  - C - Fairly reliable
+  - D - Not usually reliable
+  - E - Unreliable
+  - F - Reliability cannot be judged
+  required: true
+  type: 15
+- defaultvalue: suddenDeath
+  name: feedExpirationPolicy
+  display: ""
+  options:
+    - never
+    - interval
+    - indicatorType
+    - suddenDeath
+  required: false
+  type: 17
+- defaultvalue: '20160'
+  display: ""
+  name: feedExpirationInterval
+  required: false
+  type: 1
+- defaultvalue: '15'
+  display: Feed Fetch Interval
+  name: feedFetchInterval
+  required: false
+  type: 19
+- additionalinfo: When selected, the exclusion list is ignored for indicators from
+    this feed. This means that if an indicator from this feed is on the exclusion
+    list, the indicator might still be added to the system.
+  display: Bypass exclusion list
+  name: feedBypassExclusionList
+  required: false
+  type: 8
+- display: Trust any certificate (not secure)
+  name: insecure
+  required: false
+  type: 8
+- display: Use system proxy settings
+  name: proxy
+  required: false
+  type: 8
+description: Use the AutoFocus Feeds integration to fetch daily indicators from AutoFocus.
+display: AutoFocus Daily Feed
+name: AutoFocus Daily Feed
+feedIncremental: true
```

Comment:
Is this the correct place for this? I think it should be under `configuration`. @Nitzperetz please confirm.

Reply:
@glicht , correct, it should a configuration.
I do recommend involving @kirbles19 to create a description which should be used everytime incremental feed is used.